lib/handlers/api.js

```diff
@@ -36,6 +37,16 @@ router.post('/apps/:domain/:environment/:appname/migratedb', fhmbaasMiddleware.a
   return res.json(req.createDbResult);
 });
 
+router.get('/apps/:domain/:environment/:appname/dedicated/db', function(req, res, next) {
+  mongoUtils.hasDedicatedDb(req.params, function(err, hasDedicatedDb) {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)